test/lib/tasks/seeds_test.rb

```diff
@@ -27,7 +29,7 @@ class SeedsTest < ActiveSupport::TestCase
     def seed
       User.current = FactoryBot.build(:user, :admin => true,
                                              :organizations => [], :locations => [])
-      load Rails.root.join('db', 'seeds.rb')
+      load Rails.root.join('db/seeds.rb')
```

Comment:
Why this change? Why is it better?

Reply:
`EnforcedStyle: slashes` is the default for https://www.rubydoc.info/gems/rubocop-rails/2.5.2/RuboCop/Cop/Rails/FilePath

Reply:
That is surprising